### PR TITLE
Feat: Screen & routes

### PR DESCRIPTION
**Pull request by AshutoshM29**
Added screens and routes for the "Quiz App"

Files to check:
[App.css](src/App.css)
[App.js](src/App.js)
[Homepage](src/Pages/HomePage.jsx)
[Landing Page](src/Pages/LandingPage.jsx)
[Category card](src/Pages/Category/category-card.jsx)
[Navigation](src/components/Navigation/Navigation.jsx)
[Footer](src/components/Footer/Footer.jsx)

Files to avoid: 
BIN files
[Index](public/index.html)
[Navigation.css](src/components/Navigation/navigation.css)
[Footer.css](src/components/Footer/footer.css)

### CODE REVIEW

File: src/components/Navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    color: var(--secondary-color);
+    border-radius: 5rem;
+    justify-self: right;
+}
+
+.search-bar-input {
+    width: calc(100% + 2rem);
+    border: none;
+    background-color: transparent;
+    padding: 1rem 1rem;
```
Comment (Sahil25061999) [inline src/components/Navigation/navigation.css]: you can just use padding : 1rem here

**Review by Sahil25061999 — APPROVED** [review]
Mostly looks good to me, you can move the category card into the components folder. Also replace the anchor tags with the Link tag else its good to merge.

File: src/components/Navigation/navigation.css (hunk 1 above)
Comment (AshutoshM29) [reply src/components/Navigation/navigation.css]: > Mostly looks good to me, you can move the category card into the components folder. Also replace the anchor tags with the Link tag else its good to merge.

Thanks for the suggestion, I have fixed it

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,60 @@
...
+* {
+  --white: white;
+  
+  --main-color: #9d0208;
+  --secondary-color: #d00000;
+  --third-color: #dc2f02;
+  --forth-color: #e85d04;
+  --orange-color: #f48c06;
+  --orange-dominant: #F0590E;
+  /* --orange-mild: #faa307; */
```
Comment (varun1233) [inline src/App.css:10]: Remove commented code if not required

File: src/Pages/LandingPage.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { Link } from 'react-router-dom'
+import "../stylesheets/landing-page.css"
+import { heroLogo } from "../assets/images"
+
+function LandingPage() {
+return (
+    <>
+    <main>
+        <img className="img-hero" src={heroLogo} alt="" />
```
Comment (varun1233) [inline src/Pages/LandingPage.jsx:9]: Provide alt tag

File: src/components/Footer/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./footer.css";
+function Footer() {
```
Comment (varun1233) [inline src/components/Footer/Footer.jsx:3]: Code formatting required

**Review by varun1233 — APPROVED** [review]
Code looks good. Just added minor feedbacks.

File: src/Pages/LandingPage.jsx (hunk 3 above)
Comment (AshutoshM29) [reply src/Pages/LandingPage.jsx:9]: Thanks for pointing that out

**Pull request merged**